src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -3,6 +3,11 @@
 import java.lang.reflect.Field;
 
 import org.usfirst.frc.team2706.robot.commands.autonomous.plays.AlignAndDistance;
+import org.usfirst.frc.team2706.robot.commands.teleop.Climb;
+//import org.usfirst.frc.team2706.robot.commands.teleop.GearHandler;
+//import org.usfirst.frc.team2706.robot.commands.teleop.GearHandlerClosed;
```

Comment:
You can do this automatically once you uncomment them be pressing Ctrl + Shift + O